pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -167,7 +167,7 @@ const NET_CONFIG_TEMPLATE = `{
   "mtu": %d,
   "addIf": "%s",
   "isGateway": true,
-  "ipMasq": true,
+  "ipMasq": false,
```

Comment:
being able to route to pod ips is a requirement for kube anyway, was there a reason we set this to being with?